.babelrc-v9.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26527&commitId=bb46a1431d1d5c5ef476050365e86f1edbf04d0b&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Checked Mixed Teams Dark.chromium.png|88| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>